src/injectscripts/jobs/jobs.js

```diff
@@ -9,60 +9,18 @@ window.FinaliseSelected = function FinaliseSelected(words,beaconStringDate) { //
 }
 
 
-$("#lighthouseSummaryButton").mouseenter(function(ev){
-  summary();
-});
-
-document.getElementById("lighthouseSummaryButton").onclick = function() {
-  summary();
-}
-
-
-function summary() {
+function updateFilterParameters(event){
+  var $t = $(event.target);
   var exports = JSON.parse(filterDataForExport());
-  if (exports.hasOwnProperty("Hq")) {
-    $("#lighthouseSummaryButton").attr("href",lighthouseUrl+"pages/summary.html?host="+location.hostname+"&hq="+exports.Hq+"&start="+encodeURIComponent(exports.StartDate)+"&end="+encodeURIComponent(exports.EndDate));
-  } else {
-    $("#lighthouseSummaryButton").attr("href",lighthouseUrl+"pages/summary.html?host="+location.hostname+"&start="+encodeURIComponent(exports.StartDate)+"&end="+encodeURIComponent(exports.EndDate));
-  }
+  var page = lighthouseUrl + 'pages/' + $t.data('page') + '.html';
+  var queryString = 'host='+location.hostname+'&start='+encodeURIComponent(exports.StartDate)+'&end='+encodeURIComponent(exports.EndDate);
```

Comment:
you can break this out onto multiple lines with `\` as the last character of the line.